src/widgets/MeterPanel.cpp

```diff
@@ -346,11 +346,11 @@ MeterPanel::MeterPanel(AudacityProject *project,
       mSlider = std::make_unique<LWSlider>(this, XO(""),
          pos,
          size,
-         FRAC_SLIDER,
+         PERCENT_SLIDER,
```

Comment:
OK, after another commit that changes this default, I do see tooltips, on macOS -- but only if I click the thumb, not when I just hover the mouse over it.  This is different from the pan and gain sliders.

The text I see is like  ": 47%" suggesting something was supposed to be formatted before the ":" but is left empty.

I think that is mName being empty at these lines of ASlider.cpp:

```
      /* i18n-hint: An item name followed by a value, with appropriate separating punctuation */
      label = XO("%s: %s").Format( mName, val );
```

With pan and gain sliders, I see a hover text, but when I press the mouse, I see the same text, but it moves and is drawn in a bordered, rounded rectangle.  That's a little strange.